lib/StatusIndicator/styles.scss

```diff
@@ -2,30 +2,55 @@
    Config
    ========================================================================== */
 
-$status-indicator-size: 1em !default;
 $status-indicator-label-color--text: rgb(111, 121, 129) !default;
 
 /* ==========================================================================
    Styles
    ========================================================================== */
 
 .status-indicator {
-  display: inline-block;
-  vertical-align: middle;
   color: $status-indicator-label-color--text;
+  .due-date {
```

Comment:
I think in this case going with `.status-indicator__children` might be best, the `.due-date` was coming from the storybook example, but I'll remove that there 👍